src/helm/benchmark/presentation/summarize.py

```diff
@@ -158,6 +151,38 @@ def get_method_display_name(model_display_name: Optional[str], info: Dict[str, A
     return (model_display_name or "???") + (f" [{dict_to_str(info)}]" if len(info) > 0 else "")
 
 
+def compute_row_win_rates(table: Table) -> List[Optional[float]]:
+    """
+    Computes the average win rate of each row across columns. That is, for a given row, if we pick a random column and
+    another random row, what is the probability that is row has a better value.
+    Returns a list of average win rates, one per row, with None if a row was never meaningfully comparable.
+    """
+    win_rates_per_row: List[List[float]] = [[] for _ in table.rows]
```

Comment:
The logic is a bit cleaner as-is in my opinion since the matrix is sparse (missing values) and thus using lists feels natural (we don't need to be careful aggregating NaNs).

(it's also a simple change if we decide to do something else)

Reply:
cool, makes sense to me!